src/niscope/metadata/functions_addon.py

```diff
@@ -0,0 +1,85 @@
+# TODO(texasaggie97) update this file with all NI-FGEN function information
```

Comment:
NI-SCOPE?
You can reference #519 or remove altogether since the issue exists.

Reply:
Done